src/graphql/mirror.js

```diff
@@ -886,3 +886,54 @@ function _primitivesTableName(typename: Schema.Typename) {
   }
   return `"primitives_${typename}"`;
 }
+
+/**
+ * Convert a prepared statement into a JS function that executes that
+ * statement and asserts that it makes exactly one change to the
+ * database.
+ *
+ * The prepared statement must use only named parameters, not positional
+ * parameters.
+ *
+ * The prepared statement must not return data (e.g., INSERT and UPDATE
+ * are okay; SELECT is not).
+ *
+ * The statement is not executed inside an additional transaction, so in
+ * the case that the assertion fails, the effects of the statement are
+ * not rolled back by this function.
+ *
+ * This is useful when the statement is like `UPDATE ... WHERE id = ?`
+ * and it is assumed that `id` is a primary key for a record already
+ * exists---if either existence or uniqueness fails, this method will
+ * raise an error quickly instead of leading to a corrupt state.
+ *
+ * For example, this code...
+ *
+ *     const setName: ({|+userId: string, +newName: string|}) => void =
+ *       _makeSingleUpdateFunction(
+ *         "UPDATE users SET name = :newName WHERE id = :userId"
+ *       );
+ *     setName({userId: "user:foo", newName: "The Magnificent Foo"});
```

Comment:
Good to see a return of the Magnificent Foo Plugin

Reply:
I thought that you might enjoy that.